ts/src/bingx.ts

```diff
@@ -2658,7 +2658,7 @@ export default class bingx extends Exchange {
             if (isStopLoss || isTakeProfit) {
                 const stringifiedAmount = this.numberToString (amount);
                 if (isStopLoss) {
-                    const slTriggerPrice = this.safeString2 (stopLoss, 'triggerPrice', 'stopPrice', stopLoss);
+                    const slTriggerPrice = this.safeString2 (stopLoss, 'triggerPrice', 'stopPrice');
```

Comment:
@AresArtemius can you also restore this pls?

Reply:
restored